PointOfInterest/doc/introduction.md

```diff
@@ -2,4 +2,17 @@
 
 These data models allow to model points of interest and related entity types:
 
-+ PointOfInterest : A harmonised geographic description of a Point of Interest.
\ No newline at end of file
++ `PointOfInterest` : A harmonised geographic description of a point of interest. According to [Wikipedia](https://en.wikipedia.org/wiki/Point_of_interest)
+a point of interest, or POI, is a specific point location that someone may find useful or interesting. 
+
++ `Beach` : A harmonised description of a beach.
+[OpenStreetMap](http://wiki.openstreetmap.org/wiki/Tag:natural%3Dbeach) defines it as a
+loose geological landform along the coast or along another body of water consisting of sand, gravel,
+shingle, pebbles, cobblestones or sometimes shell fragments, etc.
+
++ `Museum` : A harmonised description of a museum.
```

Comment:
Although not really related to this PR's content (I mean dealing about beaches), I guess it is OK this new `Museum` entry in the documentation, right, right? The question also applies to the `TouristInformationCenter` just below this one :) Thanks!